### ISSUE TITLE
Get compatible with Office Excel

### ISSUE THREAD

**Issue by wangwcq** [OmbraDiFenice/table2csv#1]
https://github.com/OmbraDiFenice/table2csv/blob/42238d8c90d4d0a3ed066fcf8e33a1779919118b/src/table2csv.js#L27

Use code below instead, so as to get compatible with Office Excel.
```
    element.setAttribute('href', 'data:text/plain;charset=utf-8,\ufeff' + encodeURIComponent(text));
````

**Comment by OmbraDiFenice** [comment]
I just tried that but I don't see any difference. I can open the csv with my MS Excel in both cases the same way.
Which version of Excel are you referring to?

**Comment by suptimal** [comment]
this fixes broken "umlaut" for me in german strings (for example, München) :
https://en.wikipedia.org/wiki/Umlaut_(linguistics).

**Comment by OmbraDiFenice** [comment]
Thank you @blendi-93, that I could verify myself :)
I have added the BOM bytes to the metadata. Published on npm as v1.1.2
